app_flutter/lib/build_dashboard_page.dart

```diff
@@ -48,6 +97,10 @@ class BuildDashboardPage extends StatelessWidget {
           title: statusTable[_buildState.isTreeBuilding],
           backgroundColor: colorTable[_buildState.isTreeBuilding],
           actions: <Widget>[
+            FlatButton(
+              child: const Icon(Icons.sort),
```

Comment:
nit: Icons add vagueness to the dashboard, and it's preferred to use text (e.g. 'Filter')

Reply:
I guess I prefer them because they don't require localization, but I guess our entire build environment requires English.

In the future, I can see adding alternate sorting capabilities as well as filtering. Would "settings" be a better name? (icon?)

Reply:
SGTM. A settings icon is pretty universal and would be okay (similar to the hamburger menu icon). We could also move the branch dropdown selection into that settings area.

Reply:
I switched to the settings icon and moved the branch selector in there as well.